pkg/linter/lintBlockedTCPReposnse.go

```diff
@@ -0,0 +1,95 @@
+/*
+Copyright 2023- IBM Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package linter
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/np-guard/models/pkg/connection"
+	"github.com/np-guard/vpc-network-config-analyzer/pkg/vpcmodel"
+)
+
+const blockedTCPResponse = "blocked-TCP-response"
+const deliminator = "/"
+
+// overlapSubnets: overlapping subnet ranges (relevant mostly for the multiple VPCs use case)
+type blockedTCPResponseLint struct {
+	connectionLinter
+}
+
+// TCP connection with no response
+type blockedTCPResponseConn struct {
+	src           vpcmodel.EndpointElem
+	dst           vpcmodel.EndpointElem
+	tcpRspDisable *connection.Set
+}
+
+// /////////////////////////////////////////////////////////
+// lint interface implementation for overlapSubnets
+// ////////////////////////////////////////////////////////
+func (lint *blockedTCPResponseLint) lintName() string {
+	return blockedTCPResponse
+}
+
+func (lint *blockedTCPResponseLint) lintDescription() string {
+	return "Blocked TCP response"
+}
+
+func (lint *blockedTCPResponseLint) check() error {
+	for _, nodesConn := range lint.nodesConn {
+		for _, line := range nodesConn.GroupedConnectivity.GroupedLines {
+			tcpRspDisable := line.CommonProperties.Conn.TCPRspDisable
+			if !tcpRspDisable.IsEmpty() {
+				lint.addFinding(&blockedTCPResponseConn{src: line.Src, dst: line.Dst, tcpRspDisable: tcpRspDisable})
+			}
+		}
+	}
+	return nil
+}
+
+///////////////////////////////////////////////////////////
+// finding interface implementation for overlapSubnets
+//////////////////////////////////////////////////////////
+
+func (finding *blockedTCPResponseConn) vpc() []string {
+	return []string{getVPCFromEndpointElem(finding.src).Name(), getVPCFromEndpointElem(finding.dst).Name()}
+}
+
+func getVPCFromEndpointElem(ep vpcmodel.EndpointElem) vpcmodel.VPCResourceIntf {
+	if vpcResource, ok := ep.(vpcmodel.VPCResourceIntf); ok {
+		return vpcResource.VPC()
+	}
+	// should never get here; thus not returning an error
+	return nil
+}
+
+func (finding *blockedTCPResponseConn) string() string {
+	vpcSrc := finding.vpc()[0]
+	vpcDst := finding.vpc()[1]
+
+	srcToDstStr := fmt.Sprintf("from %v%s%s to %v%s%s",
+		vpcSrc, deliminator, finding.src.Name(), vpcDst, deliminator, finding.dst.Name())
+
+	return fmt.Sprintf("In the connection %s %s response is blocked", srcToDstStr,
+		strings.ReplaceAll(finding.tcpRspDisable.String(), "protocol: ", ""))
```

Comment:
is the `ReplaceAll` still needed ?

Reply:
you mean `ReplaceAll` as appose to `Replace`?
or any `Replace`?

Reply:
any Replace

Reply:
I think it is. Otherwise it is confusing.